cineast-core/src/main/java/org/vitrivr/cineast/core/util/CineastConstants.java

```diff
@@ -18,9 +18,18 @@
   public static final String SEGMENT_ID_COLUMN_QUALIFIER = "segmentid";
 
   /**
-   * In some legacy, adampro return "ap_distance" instead of the true distance.
-   * In an effort to unify the codebase, use this constant whenever you are expecting a distance or score from a query to the db
+   * Use this column qualifier if it might refer to an object id
+   */
+  public static final String OBJECT_ID_COLUMN_QUALIFIER = "objectid";
+
+  /**
+   * In some legacy, adampro return "ap_distance" instead of the true distance. In an effort to unify the codebase, use this constant whenever you are expecting a distance or score from a query to the db
    */
   public static final String DB_DISTANCE_VALUE_QUALIFIER = "distance";
 
+
+  public static final String DOMAIN_COL_NAME = "domain";
```

Comment:
I don't fully understand why `OBJECT_ID_COLUMN_QUALIFIER` and `DOMAIN_COL_NAME` coexist in this constants collection. I'd suggest to use either _qualifier_ or _name_ :-D